docs/standard/serialization/binaryformatter-migration-guide/winforms-applications.md

```diff
@@ -0,0 +1,99 @@
+### Drag-and-drop feature
+
+If your drag-and-drop scenario involves types that aren't intrinsically handled during serialization and deserialization, BinaryFormatter is used when [`Control.DoDragDrop`](/dotnet/api/system.windows.forms.control.dodragdrop) is called, and when [`DataObject.GetData`](/dotnet/api/system.windows.dataobject.getdata) is called to retrieve a type that's been dragged out of process. With the BinaryFormatter removal, you'll now see a string about BinaryFormatter being removed when you drop a dragged item in another process for drag-and-drop operations with types that aren't intrinsically handled.
+
+### Resources (ResX)
+
+#### The Windows Forms Designer
+
+The Windows Forms Out-Of-Process Designer also uses `BinaryFormatter` internally for ResX serialization and deserialization.
+
+Types and properties might participate in serialization without you realizing due to the standard behavior of the Windows Forms Designer. One way that BinaryFormatter is used that you might not be aware of is when a `public` property on a <xref:System.ComponentModel.IComponent> is introduced and that component is populated or that property is edited at design time. It's likely that that data gets serialized into resource files. Consider the following conditions:
+
+- A public property contains data at the time when a `Form` in the Designer is saved.
+- That property is not read-only.
+- That property is not attributed with `DesignerSerializationVisibility(false)`.
+- That property does not have a DefaultValueAttribute.
+- That property does not have a respective `bool ShouldSerialize[PropertyName]` method that returns `false` at the time of the CodeDOM serialization process. (Note: the method can have `private` scope.)
+- That property does not have a [`DesignerSerializer`](/dotnet/api/microsoft.visualstudio.modeling.dsldefinition.designerserializer)
+
+If these statements are true, the Designer determines if that property’s type has a type converter. If it does, the Designer uses the type converter to serialize the property content. Otherwise, it uses BinaryFormatter to serialize the content into the resource file.
+Windows Forms has added analyzers along with code fixes to help bring awareness to this type of behavior where BinaryFormatter serialization might be occurring without the developer’s knowledge.
+
+#### Loading resource during runtime
+
+Types that had been previously serialized into resource files via `BinaryFormatter` will continue to deserialize as expected without the need for `BinaryFormatter` as the content of ResX files are considered trusted data. In the rare case that deserialization cannot occur without `BinaryFormatter`, it can be added back for compatibility. See [BinaryFormatter migration guide: Compatibility Package](compatibility-package.md) for details. Note that an extra step of setting `System.Resources.Extensions.UseBinaryFormatter` app context switch to `true` is required to use `BinaryFormatter` for resources.
```

Comment:
@jeffhandley I believe this is about the extent that users need to know regarding embedded resources and WinForms, so I'm not sure a dedicated page is needed for this. Did we want to provide in depth details around embedded resources?

Reply:
Can you clarify what you mean by this? Are you suggesting that the entire page should be collapsed to this one section and that this section could live elsewhere?

Reply:
In the outline provided in PR description we have dedicated page for embedded resources that is assigned to Jeremy and I. In the context of WinForms, this is all the user needs to know about their embedded resources -- that we have taken care of it so that resources will still deserialize as expected without BinaryFormatter and should users run into a case where it is not deserializing as expected, the compat package needs to be pulled in. But I am not sure if what was intended for that page to begin with was to be broader than just WinForms and have more in-depth technical information.

Reply:
Ah, I see. That makes sense.

I've added a section to index.md that points to this section.